ui/app/package.json

```diff
@@ -94,7 +94,7 @@
     "@babel/runtime": "^7.8.3",
     "babel-plugin-react-intl": "^5.1.16",
     "ncp": "^2.0.0",
-    "node-sass": "^4.12.0",
+    "node-sass": "4.13.1",
```

Comment:
Let's use ^4.13.1 so we keep getting updates from dependabot. Not sure it would still send updates if an exact version is specified.

Reply:
Same change for both PRs to both branches